scanners/nmap/templates/nmap-scan-type.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           restartPolicy: OnFailure
           containers:
             - name: nmap
-              image: scbexperimental/nmap:7.80
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default 7.80 }}"
```

Comment:
Can you make this default to the charts appVersion?
Then we'd not have to change this in multiple places in the chart.